etc/preload-daemon

```diff
@@ -2,41 +2,33 @@
 
 #runs in the background and refreshes all the list files
 
+#Only one instance of this script should run at a time. If this is removed, preload will form a fairly good fork bomb.
+if [ "$(pgrep preload-daemon | wc -l)" -gt 2 ];then
```

Comment:
this confuses me... why are there two allowed now? when I moved this up in the file (in the commit that I reverted) the preload-daemon just wouldn't run for some reason...

Reply:
oh I see... you just got rid of the reloading entirely